contributing/code/core_team.rst

```diff
@@ -123,7 +114,8 @@ Symfony contributions:
 
 * **Bernhard Schussek** (`webmozart`_);
 * **Abdellatif AitBoudad** (`aitboudad`_);
-* **Romain Neutron**.
+* **Romain Neutron**;
```

Comment:
By the way, was it intended not to link to the GitHub profile here?

Reply:
I don't remember, I will ask Romain but he is on a two-week vacation right now.